src/windows/BuildInfoProxy.js

```diff
@@ -0,0 +1,18 @@
+BuildInfoProxy = {
+	init: function(successCallback, errorCallback, args) {
+        var package = Windows.ApplicationModel.Package.current;
+        var version = package.id.version;
+        var buildInfo = {
+            name: package.displayName,
+            displayName: package.displayName,
+            debug: Debug.DebugPlugin.isDebug(),
+            version: '%s.%s.%s.%s'.fmt(version.major, version.minor, version.build, version.revision),
```

Comment:
`fmt()` is our own function we inject into the String prototype in our JS. I guess this should work okay because our JS has it defined, but it would probably be safer to fall back to classic string concatenation since this is a separate code-base. (If we ever to rename/remove `fmt()` in our main JS, we would never think to look here.)